dockers/actions-plan-preview/main.go

```diff
@@ -72,6 +75,14 @@ func main() {
 		log.Printf("Successfully commented plan-preview result on pull request\n%s\n", *comment.HTMLURL)
 	}
 
+	doMinimizeComment := func(commentID githubv4.ID) {
+		err := minimizeComment(ctx, ghClientV4, commentID, "OUTDATED")
+		if err != nil {
+			log.Fatal(err)
+		}
```

Comment:
This is not a required task so I think we just log the error without stopping the process.
I mean just using `log.Printf` with a warning message is fine.